Userland/Utilities/file.cpp

```diff
@@ -122,7 +122,7 @@ static Optional<DeprecatedString> elf_details(DeprecatedString description, Depr
     __ENUMERATE_MIME_TYPE_DESCRIPTION("image/bmp", "BMP image data", image_details)                                 \
     __ENUMERATE_MIME_TYPE_DESCRIPTION("image/gif", "GIF image data", image_details)                                 \
     __ENUMERATE_MIME_TYPE_DESCRIPTION("image/jpeg", "JPEG image data", image_details)                               \
-    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/png", "PNG image data", image_details)                                 \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/webp", "WebP image data", image_details)                               \
```

Comment:
Removing PNG in favor of partial webp is a bit overeager

Reply:
:yakkie: